Common/Subworlds/RavencrestSubworld.cs

```diff
@@ -36,13 +81,16 @@ private void SpawnWorld(GenerationProgress progress, GameConfiguration configura
 		StructureTools.PlaceByOrigin("Assets/Structures/Ravencrest", new Point16(Width / 2, 188), new(0.5f));
 		Main.spawnTileX = Width / 2;
 		Main.spawnTileY = 188;
+
+		NPC.NewNPC(Entity.GetSource_TownSpawn(), Main.maxTilesX * 8, Main.maxTilesY * 8 - 200, ModContent.NPCType<BlacksmithNPC>());
+		NPC.NewNPC(Entity.GetSource_TownSpawn(), Main.maxTilesX * 8, Main.maxTilesY * 8 - 200, ModContent.NPCType<HunterNPC>());
 	}
 
 	public override void Update()
 	{
 		foreach (Player player in Main.ActivePlayers)
 		{
-			player.GetModPlayer<StopBuildingPlayer>().ConstantStopBuilding = true;
+			//player.GetModPlayer<StopBuildingPlayer>().ConstantStopBuilding = true;
```

Comment:
Was this intentional?

Reply:
I don't know why it's just commented out, but yes. Being in a `MappingSubworld` now disables building instead of only `BossDomain`s, which deprecated this code.